queries/instance_queries.sql

```diff
@@ -106,4 +106,4 @@ FROM sys.dm_os_process_memory proc_mem,
     sys.dm_os_sys_memory sys_mem,
     sys.dm_os_performance_counters perf_count WHERE object_name = 'SQLServer:Memory Manager'
 
-select @@SERVERNAME as instance_name
\ No newline at end of file
+select COALESCE( @@SERVERNAME, SERVERPROPERTY('ServerName'), SERVERPROPERTY('MachineName'), 'MSSQL Server') as instance_name
```

Comment:
this should be updated to match the query in sql_instance.go

Reply:
Fixed 👍